### ISSUE TITLE
Mouse has slight delay/kick in time, making stylus usage impossible

### ISSUE THREAD

**Issue by michaelcadilhac** [TigerVNC/tigervnc#1337]
**Describe the bug**
When clicking and dragging, there is a slight waiting time before the information is sent to the client.  Short strokes are not sent altogether.  When using a stylus, which basically translates to a sequence of short strokes, this makes interaction quite impossible.  See this video for more: https://imgur.com/sYj7AFK

**To Reproduce**
1. Connect to a VNC server
2. Start a writing app, e.g., xournalpp
3. Use stylus

**Expected behavior**
Short strokes should be sent, and mouse info should be sent as soon as the client receives it.  RealVNC, bVNC and other clients do not exhibit this behavior and work as expected.
 
**Screenshots**
See this video for more: https://imgur.com/sYj7AFK

**Client:**
 - OS: Tested on Linux, ChromeOS and Windows
 - VNC client: TigerVNC
 - VNC client version: v1.11.0
 - Client downloaded from: Linux distribution or Windows release on GitHub

**Server (please complete the following information):**
 - OS: Arch Linux
 - VNC server: TigerVNC
 - VNC server version: v1.11.0
 - Server downloaded from: Linux distribution
 - Server was started using: systemctl start vncserver

**Comment by CendioOssman** [comment]
Hmm... Odd. The only rate limiting we have is `PointerEventInterval`, but it is set very low and is even ignored if there is a button event involved. But to test, could you try giving the argument `-PointerEventInterval=0` to the client and see if that helps?

And just to clarify, did you test those other clients with the same TigerVNC server?

**Comment by CendioOssman** [comment]
It could also be that your pen is being registered as a touch device, so we're trying to detect gestures from it. Does TigerVNC 1.10.1 work better?

**Comment by michaelcadilhac** [comment]
Now _that's_ some nice guesswork!  TigerVNC 1.10.1 does work as expected!  Fantastic.  Is there an option to deactivate gesture detection, maybe on a per input device basis?

**Comment by CendioOssman** [comment]
No. There are no settings for gestures. And ideally we want to keep it that way. So I'm hoping there is a way to detect that you are using a pen, not your finger.

What does `xinput list --long` state for your device?

**Comment by michaelcadilhac** [comment]
On Google Chrome, I'm using Crostini, their newfangled Linux layer which is on Wayland.  I use an Arch Linux container.  Now ChromeOS only provides a partial simulation at this point, and `libinput` doesn't return anything.  `xinput --test-xi2` shows that touch and pen are merged into a single device `xwayland-touch`, and there seems to be no way to distinguish them (this is also a problem for native Wayland apps like xournal++).

On Windows, well, unsure how to test that, not very fluent with the system.  We use a Wacom Cintiq Pro 24, and it does look like it's treated as a touch input rather than stylus input in the eyes of vncviewer.

**Comment by CendioOssman** [comment]
If ChromeOS doesn't handle this properly natively, then the odds of getting it working remotely are unfortunately slim.

Did you have a native Linux system to test on as well?

Windows will have to be dealt with separately. They have an entirely different input system.

**Comment by michaelcadilhac** [comment]
Sorry, I'm unsure what you mean by "remotely".  vncserver is running on an arch linux PC, vncclient on arch linux over Crostini on Chrome OS.   Crostini merges the touchscreen and the stylus into one device, and indeed, using TigerVNC 1.10.1 solves the problem, but I'd rather not have to maintain a fork just to deactivate gestures.  

Most Chromebooks have a touch screen and they now all have a Linux layer, so it does not seem unreasonable to let the user decide whether they want to bypass gesture support to have their stylus working.  Can I ask why you are against having a flag to turn gesture detection off?

I do not have a native Linux system with which I can use a stylus (I'll edit the original post to reflect this).
